graphql/integration_atomic_test.go

```diff
@@ -52,7 +52,7 @@ func TestAtomicGQLQueries(t *testing.T) {
 func setup(t *testing.T, directory string) atomicGraphQLState {
 	const apiKey = "testapikey"
 	const apiUser = "testuser"
-
+	const slackUsername = "testslackuser"
```

Comment:
This should be set in the test data json file shouldn't it?

Reply:
This test relies on information for the current user which is only created over here. Im not sure if its possible to update the current users information when loading in data from the data.json file. 

https://github.com/evergreen-ci/evergreen/blob/61b608ad5036afcfd40eae6c919be16f44e8c21b/graphql/integration_atomic_test.go#L135 
The test suite looks like it only performs an `insertMany` into the db based off the information in the `data.json`. I dont think its possible to update the current users information based off this setup, not sure tho. What do you think?

Reply:
Oh yea Arjun ran into this before and I made the same comment. In real evergreen the user is retrieved from the db which is what I assumed is what happens here, however the test actually sets up a mock authentication system that does not maintain users in the db